src/oedisi/tools/cli_tools.py

```diff
@@ -156,57 +156,97 @@ def validate_optional_inputs(
             component, "container_port"
         ), f"post parameter required for component {component.name} for multi-continer model build"
 
+
+def drop_null_values(model: Any)-> dict:
+    clean_model = {}
+    assert isinstance(model, dict), "input to this function should be a dict"
+    for k, v in model.items():
+        if "_" in k:
+            key_name = k.split("_")
+            if len(key_name[1]) > 2:
+                key_name[1] = key_name[1].capitalize()
+            else:
+                key_name[1] = key_name[1].upper()
+            key_name = "".join(key_name)
+        else:
+            key_name = k    
+                
+        if isinstance(v, dict):
+            clean_model[key_name] = drop_null_values(v)
+        elif isinstance(v, list):
+            new_list = []
+            for val in v:
+                new_list.append(drop_null_values(val))
+            clean_model[key_name] =new_list
+        elif v is not None:
+            clean_model[key_name] = v
+    return clean_model
+
 def create_kubernetes_deployment(
     wiring_diagram: WiringDiagram, target_directory, broker_port, node_port
 ):
     kube_folder = os.path.join(target_directory, "kubernetes")
     if not os.path.exists(kube_folder):
         os.mkdir(kube_folder)
+    
+    SERVICE_NAME = "oedisi-service"
```

Comment:
Make into global constant instead. Maybe in types.common?

Reply:
replaceed with KUBERNETES_SERVICE_NAME. moved to common